virtcontainers/sandbox.go

```diff
@@ -1166,6 +1167,16 @@ func (s *Sandbox) CreateContainer(contConfig ContainerConfig) (VCContainer, erro
 	defer func() {
 		// Rollback if error happens.
 		if err != nil {
+			logger := s.Logger().WithFields(logrus.Fields{"container-id": c.id, "sandox-id": s.id, "rollback": true})
+
+			logger.Warning("Cleaning up partially created container")
+
+			if err2 := c.stop(true); err != nil {
```

Comment:
@evanfoster, you must change the second usage of err to err2 as well.

```suggestion
			if err2 := c.stop(true); err2 != nil {
```

Reply:
Oops!